### ISSUE TITLE
Command line reorganization

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#92]
Originally reported of Google Code with ID 36.

```
Opening bug for tracking command line reorganization
```
Originally added on 2008-07-21 13:27:59 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in 2.0
```
Originally added on 2009-04-08 13:05:15 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
